tests/infer/mcmc/test_mcmc.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+import torch
+from torch.autograd import Variable
+
+import pyro
+import pyro.distributions as dist
+from pyro import poutine
+from pyro.infer import Marginal
+from pyro.infer.mcmc.mcmc import MCMC
+from pyro.infer.mcmc.trace_kernel import TraceKernel
+from tests.common import assert_equal
+
+logging.basicConfig()
+# Change the logging level to DEBUG to see the output of the MCMC logger
+logging.getLogger().setLevel(logging.ERROR)
+
+
+class PriorKernel(TraceKernel):
+    """
+    Disregards the value of the current trace (or observed data) and
+    samples a value from the model's prior.
+    """
+    def __init__(self, model):
+        self.model = model
+        self.data = None
+
+    def setup(self, data):
+        self.data = data
+
+    def cleanup(self):
+        self.data = None
+
+    def initial_trace(self):
+        return poutine.trace(self.model).get_trace(self.data)
+
+    def sample(self, trace):
```

Comment:
Nice! This is exactly the example of stateless kernel I was thinking of when the `time_step` arg seemed weird.